### DISCUSSION TITLE
[Help]: Is there any way to remove excessive space from both left and right?

### DISCUSSION THREAD

**Discussion by Enkratia** [davidjerleke/embla-carousel#849] [Help] [answered]
### Summary

Hello, question in the title.

Video for demonstration (on the left side):
https://github.com/davidjerleke/embla-carousel/assets/74905001/bde99039-ec98-4bec-9c47-4fbfd8f29ab9

### If applicable, which variants of Embla Carousel are relevant to this question?

- [ ] embla-carousel **(Core)**
- [X] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

### CodeSandbox example

https://codesandbox.io/p/sandbox/embla-carousel-variable-widths-react-2vsfc2?file=%2Fsrc%2Fcss%2Fembla.css%3A12%2C1-15%2C48

**Comment by davidjerleke** [comment, marked as answer]
@Enkratia I think this is a duplicate of:

- #42

**Reply by Enkratia** [reply]
@davidjerleke, yes, it is. Thank you for reply.
